led-raspberrypi/web-controller-app/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"github.com/astaxie/beego"
+	controllers "github.com/kubeedge/examples/kubeedge-counter-demo/web-controller-app/controller"
```

Comment:
maybe it should be importing `github.com/kubeedge/examples/led-raspberrypi/web-controller-app/controller`